stubs/app/Http/Livewire/ResourceSelect.php

```diff
@@ -0,0 +1,54 @@
+<?php
```

Comment:
You need to run `composer format` as there are some fixable code style issues with this file.